src/pages/en/install/auto.md

```diff
@@ -33,18 +35,60 @@ yarn create astro
 pnpm create astro@latest
 ```
 
-If `create-astro` starts successfully, you will see a short list of starter templates to choose from: 
-- `starter`: A great starter template for anyone wanting to explore Astro.
-- `minimal`: A template that just includes the bare minimium to get started.
-- `blog, portfolio, docs, etc`: opinionated themes for specific use-cases.
 
-If you choose the `starter` template, you will also be asked to select which [additional frameworks](/en/core-concepts/framework-components) (React, Svelte, Vue, Solid, Preact), if any, you would like to include in your project. Additional frameworks can also be added later.
+Depending on your package manager, you may be prompted to confirm you want to install `create-astro@latest`. You will then be asked to specify a project folder (e.g. `./my-astro-site`) which will create a new directory.
+
+### Choose a Starter Template
+You will then see a short list of starter templates to choose from: 
+- `Just the basics`: A great starter template for anyone wanting to explore Astro.
+- `Blog`, `Documentation`, `Portfolio`: opinionated themes for specific use-cases.
+- `Completely empty`: A template that just includes the bare minimium to get started.
+
+Use the arrow keys (up and down) to navigate to the template you want to install, then press return (enter) to submit.
+
+> Want to check out the templates in your browser before choosing? Visit [astro.new](https://astro.new/)
+
+### Install Dependencies (optional)
+The wizard will offer to run an `install` command for you at this time, which is optional.
+
+> If you do not do so at this time, you will need to [install dependencies](/en/install/auto#2-install-dependencies) after the wizard has finished, before starting your project.
+
+### Install any Official Astro Integrations (optional)
+You will be given the option at this time to add any [additional UI frameworks](/en/core-concepts/framework-components) (React, Svelte, Vue, Solid, Preact, Lit) and then other Astro official integrations (Tailwind, Turbolinks, Partytown, Sitemap) by running `astro add --yes`
+
+To select which Astro integrations, if any, you would like to include in your project, use the arrow keys (up and down) to navigate and the space bar to toggle selected/unselected. You can select multiple items at once, or you can continue without selecting any items.
+
+
+When you are satisfied with your selection, press return (enter) to submit. 
+
+>🟢 Selected integrations will display a solid (green) circle. (All integrations are **unselected by default**.) 
+>
+>⚠️ If every item is unselected, then no integrations will be added at this time. 
```

Comment:
Sorry to be obtuse... but this feels like a bit _too_ much detail. I feel like we kinda failed if our UI needs this much explanation 😆 Would it be fair to remove this?

Reply:
that is some acute reasoning

Reply:
We did just have a request for pretty much this. . . so I put it in, because it's easier for y'all to tell me to take it out!

Reply:
I think I can tidy the rest up on my own, but I would like us to decide together if this superflous (but pretty! and green!) bit stays in. 

I'm happy to go with the will of the group, but if left to my own devices, I'd keep it in until I changed my mind. 
@bholmesdev wants it out. 
@aFuzzyBear made a joke without explicitly stating a preference, 
and I'd like @delucis to weigh in.

While I'm waiting for this to resolve itself, I'll make everyone else's edits to taste. Thanks again, everyone!! :heart:

Reply:
I’m torn on this. When I first saw this I also thought this might be more detail than necessary, but I’ve also seen a few people in support tripped up by space vs enter and what is selected/unselected. So maybe the problem _is_ with the UI? 😅 

Is there a prompt after this selection step? Something like “You didn’t select any integrations? Is that correct? (y/n)” or “You selected react and turbolinks. Is that correct? (y/n)”.

That would give users a chance to double back and make this specific advice less important.

Reply:
If I was to sine off on this off, it would be to have it removed. Simply because its opposite to the hypotenuse of dx of the CLI tool.

Reply:
I am content to have it be recorded in the annals of commit history that I went down fighting for the confused. Taking it out now.